db/managed_iterator.cc

```diff
@@ -101,9 +101,7 @@ void ManagedIterator::SeekToLast() {
   }
   assert(mutable_iter_ != nullptr);
   mutable_iter_->SeekToLast();
-  if (mutable_iter_->status().ok()) {
```

Comment:
This made no sense: if status is not ok, it would leave `valid_` containing leftovers from previous iterator operations.